src/shell.rs

```diff
@@ -28,54 +31,74 @@ impl<'a> Shell<'a> {
 mod tests {
 
     use std::path::PathBuf;
+    use std::rc::Rc;
+
+    use config::mock::MockConfig;
+
     use super::Shell;
 
     fn root_path(path_str: &str) -> PathBuf {
         PathBuf::from(path_str)
     }
 
+    fn std_mock_config() -> Rc<MockConfig> {
```

Comment:
yup.  Needed another name.  Though honestly this method isn't really even worth it.  Probably should have just used the method with argument.